src/System.ComponentModel.Annotations/tests/ValidatorTests.cs

```diff
@@ -750,7 +750,7 @@ public class ValidClassAttribute : ValidationAttribute
             protected override ValidationResult IsValid(object value, ValidationContext _)
             {
                 if (value == null) { return ValidationResult.Success; }
-                if (value.GetType().Name.ToLower().Contains("invalid"))
+                if (value.GetType().Name.ToLowerInvariant().Contains("invalid"))
                 {
```

Comment:
We should consider adding a Contains that takes StringComparision options. However given what we have I wonder if .IndexOf("invalid", StringComparision.CultureCultureIngnoreCase) < 0 is better. @tarekgh?